### PR TITLE
Use CocoaPods for PSPDFKit :(

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: none
affects: student, teacher
release note: none
test plan: builds should pass

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a reference to a jira ticket. For example: `refs: MBL-10023`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add which apps this change affects. Example: `affects: Teacher, Student` or `affects: none`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a release note. If no release note is wanted, use `none`. Example: `release note: Fixed a bug that prevented users from enjoying the app.`
  </td>
    </tr>
  </tbody>
</table>



<details><summary>:x: JS tests</summary>


    Summary of all failing tests
     FAIL  src/redux/middleware/__tests__/persist.test.js
      ● persistMiddleware › persists into async storage
    
        Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
    
        Check the render method of `Screen`.
    
          at createFiberFromTypeAndProps (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:14410:21)
          at createFiberFromElement (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:14433:15)
          at reconcileSingleElement (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:4982:23)
          at reconcileChildFibers (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:5042:35)
          at reconcileChildren (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7241:28)
          at finishClassComponent (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7634:5)
          at updateClassComponent (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:7568:24)
          at beginWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:9043:16)
          at performUnitOfWork (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12649:12)
          at workLoopSync (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:12622:22)
    
    
    Test Suites: 1 failed, 258 passed, 259 total
    Tests:       1 failed, 2148 passed, 2149 total
    Snapshots:   473 passed, 473 total
    Time:        171.075s
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/51318320/p/de5116e700bdb58cd14170f869f6153a">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/51318336/p/084166df07ea766eea1040dc30257329">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/51318297/p/8a7adc64a2b826784f339a4a8bc85fa4">
    
  </a>
</details>

File: Student/Student.xcodeproj/xcshareddata/xcschemes/CITests.xcscheme [hunk 1]
```diff
@@ -3,7 +3,7 @@
    LastUpgradeVersion = "1200"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "NO"
+      parallelizeBuildables = "YES"
```
Comment (vanwagonet) [inline Student/Student.xcodeproj/xcshareddata/xcschemes/CITests.xcscheme:6]: We has switched this to NO because it worked around the RN issues.

**Review by vanwagonet — CHANGES_REQUESTED** [review]

**Review by vanwagonet — APPROVED** [review]
If the build works, this is good to go.

**Comment by vanwagonet** [comment]
the tests all passed, I have no idea why the step seems to think it failed. I say merge anyway.

**Comment by armstrongnate** [comment]
That is really weird about the build. I, also, don't see why it says it failed.

**Pull request merged**